dbms/src/Flash/Executor/toRU.h

```diff
@@ -19,6 +19,10 @@
 namespace DB
 {
 using RU = UInt64;
+
+// Convert cpu time nanoseconds to cpu time millisecond, and round up.
+UInt64 toCPUTimeMillisecond(UInt64 cpu_time_ns);
+
 // Convert cpu time nanoseconds to Request Unit.
 RU toRU(UInt64 cpu_time_ns);
```

Comment:
Is RU actually a double?
Was it a bug to use UInt64 before? Do we need to fix it in this pr?

Reply:
I think this pr use UInt64 is still ok.
Next pr need to update to double, because next pr will report RU usage to PD, and the pb between LAC and PD use double type for RU consumption.

Reply:
ok